src/ni/vsbuild/Pipeline.groovy

```diff
@@ -88,8 +88,14 @@ class Pipeline implements Serializable {
          // need to bind the variable before the closure - can't do 'for (version in lvVersions)'
          def lvVersion = version
 
+         String nodeLabel = lvVersion
+         if (pipelineInformation.nodeLabel?.trim()){
+           nodeLabel = "$nodeLabel && ${pipelineInformation.nodeLabel}"
```

Comment:
The indentation for this line is off. We use 3 spaces for a tab, but this one is only indented 2 spaces from the previous line. The closing } is also off.